action/jsonapi.go

```diff
@@ -24,6 +26,10 @@ type queryMessage struct {
 	Query string `json:"query"`
 }
 
+type queryHostMessage struct {
```

Comment:
I start to wonder if we should move the implementation of the `jsonapi` to it's own package, e.g. `github.com/justwatchcom/gopass/utils/jsonapi` and only invoke it from this place?

Reply:
You mean invoke the jsonapi from `actions/jsonapi.go`? Sounds good - that would allow to easily split up the current stuff in `jsonapi.go` into multiple files. E.g. extracting the message struct spec.

Reply:
see latest commit. Thanks for the suggestion, it's much better now 👍